x-pack/plugins/infra/public/pages/metrics/settings/indices_configuration_panel.tsx

```diff
@@ -85,6 +105,42 @@ export const IndicesConfigurationPanel = ({
           {...metricAliasFieldProps}
         />
       </EuiFormRow>
+      {remoteClustersExist && !metricIndicesExist && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiCallOut
+            size="s"
+            title={METRIC_INDICES_WARNING_TITLE}
+            color="warning"
+            iconType="warning"
+            data-test-subj="infraIndicesPanelSettingsWarningCallout"
+          >
+            <FormattedMessage
+              id="xpack.infra.sourceConfiguration.metricIndicesDoNotExist"
+              defaultMessage="We couldn’t find any metrics data because the pattern entered doesn’t match any index."
+            />
+          </EuiCallOut>
+        </>
+      )}
+      {!remoteClustersExist && !metricIndicesExist && (
+        <>
+          <EuiSpacer size="s" />
+          <EuiCallOut
+            data-test-subj="infraIndicesPanelSettingsDangerCallout"
+            size="s"
+            title={REMOTE_CLUSTER_ERROR_TITLE}
+            color="danger"
+            iconType="error"
+          >
+            <FormattedMessage
+              id="xpack.infra.sourceConfiguration.remoteClusterConnectionDoNotExist"
+              defaultMessage="We couldn’t find any metrics data because the pattern entered doesn’t match any index."
+            />
+            Check that the remote cluster is available or that the remote connection settings are
```

Comment:
**nit:** what is this text about? Should we have only the `FormattedMessage` above?

Reply:
Thank you for spotting that - it should be inside the formatted message - forgot to replace it 🤦‍♀️ fixed 👍